### PR TITLE
fix twitter link

### PR DESCRIPTION
**Pull request by robertkowalski**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,7 +114,7 @@ <h2 class="icon icon-contribute">Want to Contribute?</h2>
...
+    <p>Because a healthy community is our most valuable asset, this is a good place to start contributing. Learn how CouchDB is used and why people love it. <strong>Read about it, promote it, and talk about it.</strong> Arrange or attend a <a href="https://cwiki.apache.org/confluence/display/COUCHDB/Meetups">meetup</a>, give a talk, or write about your adventures with CouchDB. If you write a blog post about CouchDB (or find a blog post worth sharing) drop a mail to the <a href="mailto:marketing@couchdb.apache.org">marketing list</a> (or <a href="https://twitter.com/CouchDB">send a tweet</a>) and we&rsquo;ll include it in the <a href="http://blog.couchdb.org/">weekly news</a>. You could also help out on the <a href="#user-mailing-list">user mailing list</a>, on <a href="irc://irc.freenode.net/couchdb">IRC</a>, or on <a href="http://stackoverflow.com/questions/tagged/couchdb">StackOverflow</a>. Or come join us on the <a href="#developer-mailing-list">developer mailing list</a> and let us know how else you&rsquo;d like to help.
```
Comment (kxepal) [inline index.html:117]: We need to get rid these long lines and wrap them with 80 right margin.
It's too hard to find your changes in GH /:

**Comment by kxepal** [comment]
+1

File: index.html (hunk 1 above)
Comment (janl) [inline index.html:117]: yeah!

**Comment by janl** [comment]
No need to wait for a review on these small patches, just merge :)

**Comment by robertkowalski** [comment]
got fixed with the new contribution section :)

**Pull request closed without merging**